test/unit/seed/seeder.js

```diff
@@ -301,6 +300,7 @@ describe('Seeder._listAll', function () {
             '/test/integration/seed/2-seeds/recursive-folder/ls-seed.ls',
           process.cwd() +
             '/test/integration/seed/2-seeds/recursive-folder/ts-seed.ts',
+          process.cwd() + '/test/integration/seed/2-seeds/ts-seed.ts',
```

Comment:
Why change the order?

Reply:
Comments of `Seeder._listAll`:
https://github.com/knex/knex/blob/bc04b7ab30f152a677de53e2cc6e7df995a12f98/lib/seed/Seeder.js#L51-L52

And I thought `sort` means:
1. Same order as `seeds.directory` property in `knexfile.js` (process in `Seeder._listAll`)
2. Sub-ordered by alphabet (include nested files) (process in `getFilepathsInFolder`)

Test code is violate this rule, so I change order.